.gitignore

```diff
@@ -102,3 +102,7 @@ config/AppliedEnergistics2/VersionChecker.cfg
 config/astralsorcery/amulet_enchantments.cfg
 config/brandon3055/contributors.json
 config/brandon3055/Custom Fusion Recipe Info.txt
+config/vanillafix.cfg
+config/vanillafix/config_version
+config/vanillafix/methods-stable_39.csv
+config/splash.properties
```

Comment:
we need that in I'm afraid, it configures the startup screen.

Reply:
Actually not sure when I added that to .gitignore, but apparently I did :P